test/c/basic/sign_cast.c

```diff
@@ -0,0 +1,13 @@
+#include <limits.h>
+#include "smack.h"
```

Comment:
Ah, formatting check is failing. include smack.h should be before limits.h it seems.